changelog/switch-skip-declaration.dd

```diff
@@ -0,0 +1,21 @@
+The deprecation for `switch` cases which skip the declaration of a variable has ended
```

Comment:
Period at the end of this sentence.

Reply:
This is the title - it's a git commit message like format. Check DAutoTest.

Reply:
Ah, right.  Sorry.